Source/Extensions/Blazorise.Video/VideoSource.cs

```diff
@@ -64,14 +61,23 @@ public VideoSource( string[] sources )
         return new( source );
     }
 
+    /// <summary>
+    /// Implicit converted for the array of string source.
+    /// </summary>
+    /// <param name="sources">Multiple source addresses.</param>
+    public static implicit operator VideoSource( string[] sources )
+    {
+        return new( sources );
+    }
+
     #endregion
 
     #region Properties
 
     /// <summary>
     /// True if the source contains the single address.
     /// </summary>
-    public bool Indexed { get; private set; }
+    public bool Indexed => Medias?.Count > 1;
```

Comment:
True if source contains single address?

What is this for, and why not > 0?

Reply:
Because if there is more than 1 element, it is considered as an array, and it will be serialized as an array. Otherwise, it is serialized as a primitive type.

Reply:
Maybe let's explain that? 
Maybe IsArray better conveys that meaning?

Reply:
Or `SerializeAsArray`